programs/cleos/httpc.cpp

```diff
@@ -128,6 +129,15 @@ namespace eosio { namespace client { namespace http {
    request_stream << "\r\n";
    request_stream << postjson;
 
+   if ( verbose ) {
+      string s(request.size(), '\0');
+      buffer_copy(boost::asio::buffer(s), request.data());
+      std::cerr << "REQUEST:" << std::endl
+                << "---------------------" << std::endl
+                << s << std::endl
```

Comment:
We should print the `fc::json::to_pretty_string` instead of `fc::json::to_string`